### PR TITLE
[ new ] ⁻¹-anti-homo-(// | \\)

### PR DESCRIPTION
**Pull request by gallais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -213,6 +213,7 @@ Additions to existing modules
   ⁻¹-selfInverse  : SelfInverse _⁻¹
   \\≗flip-//⇒comm : (∀ x y → x \\ y ≈ y // x) → Commutative _∙_
   comm⇒\\≗flip-// : Commutative _∙_ → ∀ x y → x \\ y ≈ y // x
+  ¹-anti-homo‿- : (x - y) ⁻¹ ≈ y - x
```
Comment (Taneb) [inline CHANGELOG.md:216]:
```suggestion
  ⁻¹-anti-homo‿- : (x - y) ⁻¹ ≈ y - x
```

**Review by Taneb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jamesmckinna) [reply CHANGELOG.md]: Suggested change: thus Jemma as stated moves to `Algebra.Properties.AbelianGroup`, as a renaming of a correctly stated lemma in `...Group`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -213,6 +213,7 @@ Additions to existing modules
   ⁻¹-selfInverse  : SelfInverse _⁻¹
   \\≗flip-//⇒comm : (∀ x y → x \\ y ≈ y // x) → Commutative _∙_
   comm⇒\\≗flip-// : Commutative _∙_ → ∀ x y → x \\ y ≈ y // x
+  ⁻¹-anti-homo‿- : (x - y) ⁻¹ ≈ y - x
```
Comment (jamesmckinna) [inline CHANGELOG.md:216]:
```suggestion
  ⁻¹-anti-homo-// : (x // y) ⁻¹ ≈ y // x
```

File: src/Algebra/Properties/Group.agda [hunk 3]
```diff
@@ -126,6 +126,14 @@ inverseʳ-unique x y eq = trans (y≈x\\z x y ε eq) (identityʳ _)
...
   x ∙ y ∙ (y ⁻¹ ∙ x ⁻¹) ∎
 
+⁻¹-anti-homo‿- : ∀ x y → (x - y) ⁻¹ ≈ y - x
+⁻¹-anti-homo‿- x y = begin
+  (x - y) ⁻¹       ≡⟨⟩
+  (x ∙ y ⁻¹) ⁻¹    ≈⟨ ⁻¹-anti-homo-∙ x (y ⁻¹) ⟩
+  (y ⁻¹) ⁻¹ ∙ x ⁻¹ ≈⟨ ∙-congʳ (⁻¹-involutive y) ⟩
+  y ∙ x ⁻¹         ≡⟨⟩
+  y - x ∎
+
```
Comment (jamesmckinna) [inline src/Algebra/Properties/Group.agda:136]:
```suggestion
⁻¹-anti-homo-// : ∀ x y → (x // y) ⁻¹ ≈ y // x
⁻¹-anti-homo-// x y = begin
  (x // y) ⁻¹       ≡⟨⟩
  (x ∙ y ⁻¹) ⁻¹    ≈⟨ ⁻¹-anti-homo-∙ x (y ⁻¹) ⟩
  (y ⁻¹) ⁻¹ ∙ x ⁻¹ ≈⟨ ∙-congʳ (⁻¹-involutive y) ⟩
  y ∙ x ⁻¹         ≡⟨⟩
  y // x ∎

```

File: CHANGELOG.md (hunk 2 above)
Comment (jamesmckinna) [reply CHANGELOG.md:216]: Just getting the hang of actually making suggestions!

**Review by jamesmckinna — APPROVED** [review]
All looks good to me, thanks for taking my suggestions!
As for renaming the property in `Algebra.Properties.AbelianGroup`, you could add that to this PR, or do that downstream... I'd be happy either way.

**Comment by gallais** [comment]
Sorry, I missed that.
Should be fixed now.

**Comment by jamesmckinna** [comment]
> I have left the re-exports mentioning `\\` and `//` just so that if people upgrade their assumptions from e.g. `Group` to `AbelianGroup`, lemmas do not mysteriously disappear.

Fair enough, though I'm probably more hard-line in wanting to insist that the 'correct' names be used... but I'll relent for once ;-)

**Review by Taneb — APPROVED** [review]

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**